Collections/Map Utilities/map.alias

```diff
@@ -4,17 +4,19 @@ testVersion = False
 argList = &ARGS&
 
 # Comment the first line for `!map`, second line for `!move`
-# args = argparse((['-t',combat().me.name if combat().me else combat().current.name] if combat() and (combat().me or combat().current) else []) + ['-move']+argList if argList else [])
+# args = argparse((['-t',combat().me.name if combat().me else combat().current.name] if combat() and (combat().me or combat().current) else []) + ['-move']+args if args else [])
```

Comment:
I think this is a confusion on my part since to test my alias I still have to use tembed as well as when trying to get the PR up to date in code after you made some changes